molecule/custom/molecule.yml

```diff
@@ -22,7 +22,7 @@ provisioner:
           enabled: true
           only_security: true
           reboot: false
-          reboot_from_time: "2:00"
+          reboot_from_time: 2:00
```

Comment:
Doesn't this change break the tests @konstruktoid ? At least for me, if I put it as something else than string, it does not go through.